doc/start/os-recommendations.rst

```diff
@@ -26,8 +26,9 @@ Linux Kernel
   for kernel version guidance.
 
   Older kernel client versions may not support your `CRUSH tunables`_ profile
-  or other newer features of the Ceph cluster, requiring the storage cluster
-  to be configured with those features disabled.
+  or other newer features of the Ceph cluster, requiring the storage cluster to
+  be configured with those features disabled. For RBD, a kernel of version 5.4
```

Comment:
I have checked and it's actually 5.3, not 5.4.  Anthony had 5.4 in mind probably because it's a major LTS kernel:
```suggestion
  be configured with those features disabled. For RBD, a kernel of version 5.3 or CentOS 8.2
```

Reply:
CentOS 8 is based on 4.18 kernel but major RBD changes from 5.3 were backported there.